src/markdoc/partials/api/console/role-response-parameters.md

```diff
@@ -18,6 +18,10 @@ The description of the role.
 Determine if the role is a super role. Super roles cannot be removed.
 {% /parameter %}
 
+{% parameter name="permissions" type="object" %}
+An object defining the permissions associated with the role. Each key in the object represents a specific permission, and the value is a boolean indicating whether the permission is granted (true) or denied (false). Permissions control access to various features and functionalities within the system.
```

Comment:
```suggestion
An object defining the permissions associated with the role. Each key in the object represents a specific permission, and the value is a boolean indicating whether the permission is granted (true) or denied (false).
```

We don't need that last sentence because it is already explained in the first part.